src/styles/components/header/header.module.scss

```diff
@@ -7,7 +8,6 @@
     background-color: var(--color-maroon);
     height: 226px;
```

Comment:
you can add 226px as a variable in globals.scss and then import it here as a var. actually name this one as desktop-header-height bc we'll need to define mobile-header-height too.